crates/common/common_config/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "tedge_config"
+name = "common_config"
```

Comment:
I'm not convinced by this changed that sounds to me quite useless and somehow unclear.

* Useless: I see only tedge related stuff under this crate.
* Unclear: common to what?

Reply:
For me "tedge_config" really sounds just for the things related our `tedge config` CLI. Since "system_services" is not a part of the `tedge config` CLI, I wanted to avoid the name "tedge_config" as the top directory name.

But I just come up with the proposal something in the middle. How about this?

```
common/
├─ tedge_config/
│  ├─ src/
│  │  ├─ tedge_config_cli/
│  │  ├─ system_services/
```

Reply:
> But I just come up with the proposal something in the middle. How about this?
```
common/
├─ tedge_config/
│  ├─ src/
│  │  ├─ tedge_config_cli/
│  │  ├─ system_services/
```


Yes, this is clear.

Reply:
@initard Can you move to this new proposal from @rina23q ? Thank you.

Reply:
Sure